app/webpack/components/authentication/sign_in.vue

```diff
@@ -242,9 +262,13 @@ export default {
           terms_accepted: termsAccepted
         },
         { headers: { 'X-CSRF-TOKEN': this.csrfToken } }
-      ).then(result => {
-        return this.afterServerSignIn(result.data);
-      });
+      )
+        .then(result => {
+          return this.afterServerSignIn(result.data);
+        })
+        .catch(() => {
+          return this.afterFailedSignIn();
```

Comment:
Suggestion: rename `afterFailedSignIn` --> `afterFailedServerSignIn` (to match the flow, and make more specific and clear).

Reply:
Done!